src/main/java/com/vailsys/freeclimb/api/call/CreateCallRequest.java

```diff
@@ -16,21 +16,20 @@
 class CreateCallRequest {
 	public String to;
 	public String from;
-
 	public String applicationId;
-
 	public CallOptions options;
 
 	/**
-	 * Setup a new call creation payload using an applicationId.
+	 * Setup a new call creation payload.
 	 *
 	 * @param to            The number to call out to (DNIS). This can be any valid
 	 *                      phone number formatted in E.164 format.
 	 * @param from          The number to call from (ANI). This must be a number
 	 *                      purchase from FreeClimb or a verified phone number owned
 	 *                      by the user.
 	 * @param applicationId The {@code applicationId} for the registered FreeClimb
-	 *                      application which should handle this call.
+	 *                      application which should handle this call. Required if
```

Comment:
It is my understanding that the FreeClimb API requires an `applicationId` field for this endpoint in all cases except when providing a `parentCallId`. It's just not required that the application have a `callConnectUrl` set, if one is provided in the request via the `CallOptions`.

If you intend to make the `applicationId` field optional in the constructor of a `CreateCallRequest` (to support the case that the SDK user intends to populate the `parentCallId` field), it should become part of the `CallOptions` type.

Reply:
Note that this may lead to user confusion if they expect the `CallOptions` to be fully optional and call creation to be possible using only required fields.